testprojects/src/scala/Adder.scala

```diff
@@ -0,0 +1,6 @@
+package org.pantsbuild.scala
+
+object Adder {
+  def add(x: Integer, y: Integer): Integer = x + y
+}
+
```

Comment:
Purely a convention for this repo, but: let's define this in the matching filepath:
```
 .../org/pantsbuild/scala/Adder.scala
```
... and probably give it a better package name.

Alternatively, maybe skip adding this and just use `testprojects/src/jvm/org/pantsbuild/example/app` instead?

Reply:
Agreed; I'll just use the existing example code. Removed.